docs/advanced-scenarios.md

```diff
@@ -185,6 +190,8 @@ Luigi allows you to implement and configure feature toggles. They can be used to
       visibleForFeatureToggles: ['ft1']
   }
   ```
+  If you define a list of multiple feature toggles the node will be restricted and it will be shown only if **all** of the specified feature toggles are set.
```

Comment:
```suggestion
  If you define a list of multiple feature toggles, the node will be restricted and it will be shown only if **all** of the specified feature toggles are set.
```